components/footers.js

```diff
@@ -45,7 +45,8 @@ export function Footer({ recentPost }) {
             <FooterLink href="/startups/">How We Work with Startups</FooterLink>
             <FooterLink href={recentPost.id}>Our Latest Post</FooterLink>
             <FooterLink href="https://donate.free.law/np/clients/freelawproject/subscribe.jsp?subscription=9">
-              Join our Newsletter
+              Join our Newsletter</FooterLink>
+            <FooterLink href="/librarians/">CourtListener Feature Guide
```

Comment:
I think I'd do:

```suggestion
            <FooterLink href="/librarians/">Librarian Guide
```

And put it under the "How we work with startups"?